JurassicWorldCollection.java

```diff
@@ -0,0 +1,38 @@
+
+package mage.sets;
+
+import mage.cards.ExpansionSet;
+import mage.constants.Rarity;
+import mage.constants.SetType;
+
+/**
+ * @author Susucr
+ */
+public final class JurassicWorldCollection extends ExpansionSet {

```

Comment:
You should not be adding a new file. There is an existing file, you only need to add the line(s) corresponding to your newly implemented card(s) to it (keep the list in alphabetical order).

Did `gen-card.pl` not automatically do this?

Reply:
It did, but ... I've reached a point where I have no idea what I am doing, and i'm not getting less confused the more I try to follow these instructions...

Reply:
In IntelliJ, there should be a "commit" tab on the right sidebar, listing all files that have changed locally, and you can select those files to add to the commit and specify a useful commit message.

In the bottom right corner, you can select the active branch you are working on and make a new one when needed.

Reply:
I am fundamentally missing something about committing in github and intelliJ. Although I understand the general concept. Here are the ways I have tried to do commits so far:
1. In intelliJ: I use the commit dialogue on the left side, and check the boxes for the files that have changed/been added, and click commit and push. I don't see any evidence of this commit or push anywhere on github, although I do see it in my list on the right hadn side. I haven't selected a branch, a user, or a path, but I can't figure out which branch I should be selecting here anyway - there's Head, master, origin/master, or I can put in something else, and I'm not sure which I should be using. Since I didn't see anything show up in my github when I looked at it online, I tried option 2.
2. I went to the my github, skaspels/mage, added a new file with the changes that i made (which is why there is a new file for the REX set), and I was able to create a pull request from there into the master, but then I have the problem that I am uploading a file for things i've changed instead of submitting changes. The instructions for pull requests on github leave me more confused because they assume I have a level of knowledge already that is rather too generous for me.